apis/notificationapi/notificationapi.yaml

```diff
@@ -1,12 +1,16 @@
 openapi: 3.0.1
 info:
   title: Notification API(s)
-  description: 'The Email resources provides the notification mechanism for all the
-    users on Sunbird. The Resource notifies users officially about something on Sunbird. '
+  description: The Email resources provides the notification mechanism for all the
```

Comment:
@sknirmalkar89 The description of this microservice is not understandable. Could you please clarify? What is meant by 'something'? To write it differently, please answer the following questions about the Notifications microservice:
1. Why do you need this microservice?
2. What does the microservice do?
3. When can one use this microservice?

Reply:
updated.